integration-tests/helpers/start-server-with-environment.js

```diff
@@ -0,0 +1,28 @@
+const setupServer = require("./setup-server")
+const { initDb } = require("./use-db")
+
+const startServerWithEnvironment = async ({ cwd, verbose, env }) => {
+  if (env) {
+    Object.entries(env).forEach(([key, value]) => {
+      process.env[key] = value
+    })
+  }
```

Comment:
thought: highlighting this again https://github.com/medusajs/medusa/pull/1723#discussion_r912374357

Reply:
that doesn't seem to delete the environment. I have opted for this solution instead where the keys of the passed along environment are deleted from `process.env`. 
```
Object.entries(env).forEach(([key, value]) => {
    delete process.env[key]
  })
```

Reply:
Got it that works as well! :)